src/o365/graph/commands/teams/teams-clone.ts

```diff
@@ -0,0 +1,208 @@
+import * as request from 'request-promise-native';
+import auth from '../../GraphAuth';
+import Utils from '../../../../Utils';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import { CommandOption, CommandValidate } from '../../../../Command';
+import GraphCommand from '../../GraphCommand';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  teamId: string;
+  displayName: string;
+  mailNickname: string;
```

Comment:
@sprider , could you please remove the `--mailNickname "libassist"` from the doc since we would drop it until the graph guys fix it.

For more info see: https://github.com/pnp/office365-cli/pull/939#discussion_r279938818

Reply:
@VelinGeorgiev, This is complete.